src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -48,6 +48,18 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", ZonedDateTime.now());
+        body.put("message", ex.getMessage());
+        body.put("status", HttpStatus.BAD_REQUEST.value());
+
+        return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
```

Comment:
The error response is not fully descriptive. It is recommended to include additional information such as path, method, and parameters in the error response.